browser/brave_content_browser_client.cc

```diff
@@ -347,3 +334,54 @@ GURL BraveContentBrowserClient::GetEffectiveURL(
   return url;
 #endif
 }
+
+// [static]
+bool BraveContentBrowserClient::HandleURLOverrideRewrite(GURL* url,
+    content::BrowserContext* browser_context) {
+  // redirect sync-internals
+  if (url->host() == chrome::kChromeUISyncInternalsHost ||
+      url->host() == chrome::kChromeUISyncHost) {
+    GURL::Replacements replacements;
+    replacements.SetHostStr(chrome::kChromeUISyncHost);
+    *url = url->ReplaceComponents(replacements);
+    return true;
+  }
+
+  // no special win10 welcome page
+  if (url->host() == chrome::kChromeUIWelcomeWin10Host ||
+      url->host() == chrome::kChromeUIWelcomeHost) {
+    *url = GURL(chrome::kChromeUIWelcomeURL);
+    return true;
+  }
+
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
+  if (url->SchemeIs(content::kChromeUIScheme) &&
+      url->host() == ethereum_remote_client_host) {
+    if (browser_context) {
+      extensions::ExtensionService* service =
+        extensions::ExtensionSystem::Get(browser_context)->extension_service();
+      extensions::ComponentLoader* loader = service->component_loader();
+      if (loader->Exists(ethereum_remote_client_extension_id)) {
+        *url = GURL(ethereum_remote_client_base_url);
+        return true;
+      }
+    }
+  }
+#endif
+
+  return false;
+}
+
+std::vector<std::unique_ptr<content::NavigationThrottle>>
+BraveContentBrowserClient::CreateThrottlesForNavigation(
+    content::NavigationHandle* handle) {
+  std::vector<std::unique_ptr<content::NavigationThrottle>> throttles =
+      ChromeContentBrowserClient::CreateThrottlesForNavigation(handle);
+
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
```

Comment:
BRAVE_WALLET_ENABLED + ENABLE_EXTENSIONS

Reply:
will be handled instead via brave_wallet_enabled = !is_official_build && enable_extensions in the gni file